<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html lang="en-US" xml:lang="en-US" xmlns="http://www.w3.org/1999/xhtml">

<head>

<link rel="stylesheet" type="text/css" href="http://gtpb.igc.gulbenkian.pt/bicourses/GTPBstyle.css">
</link>

<title>
GTPB: PW12 home
</title>

</head>

<body>

<!-- GTPB header -->

<table width="700" align="center" border="0">
<tr>
<td>
<p><img src="../images/GTPBlogo.png"></img></p>
</td>
</tr>
</table>


<!---      --->


<table width="700" align="center" border="1">
<tr><td>
<br />
<font size="6" color="#006400"><b>PW12</b></font>
<br /><h1>Automated and reproducible analysis of NGS data</h1>
</td></tr>
</table>

<br />

<table width="700" align="center" border="1">
<font size="2">
<tr><td>
<p><b>&nbsp;&nbsp;IMPORTANT DATES for PW12</b>
<br />&nbsp;&nbsp;&nbsp;Deadline for applications: <b>August 27nd 2012</b>
<br />&nbsp;&nbsp;&nbsp;Notification of acceptance dates:
<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>EARLY:</b> August 27nd 2012
<br />&nbsp;&nbsp;&nbsp;<i>(on special request, see 
<a href="application.html"><b>Application</b></a>)</i>
<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>NORMAL:</b> September 3rd 2012 
<br />&nbsp;&nbsp;&nbsp;Course date: <b> September 11th - September 14th 2012</b>
</p>
</td></tr>
</font>
</table>


<table width="700" align="center" border="1">
<tr><td>
<h2>Instructors:</h2>
</td></tr>

<tr><td>
<img border="0" style="margin:4px" src="../images/rutger.jpg" width="91"  align="left"></img> 
<font size="2"><p align="justify">
<b>Rutger Vos</b> studied biology at the University of Amsterdam, where he graduated in
2000. He then embarked on his PhD research under professor Arne Mooers at Simon
Fraser University in Vancouver, Canada, where he defended his thesis on
phyloinformatic problems in 2006. As a self-taught programmer he then became
involved in several open-source scientific software development projects while
continuing his research career through a postdoctoral fellowship at the
University of British Columbia (Vancouver, Canada) and a Marie Curie research
fellowship at the University of Reading (Reading, UK). In Spring of 2012 he
commenced his employment as the bioinformaticist of the Naturalis Biodiversity
Center in a role where he combines novel research with
bioinformatics contributions to various research programmes within the
organization. In his spare time he also contributes to various open source
software projects (<a href="http://treebase.org">TreeBASE</a>, <a
href="http://search.cpan.org/dist/Bio-Phylo">Bio::Phylo</a>, <a
href="http://nexml.org">NeXML</a>) and is co-PI of the
PhyloTastic project. In addition Rutger has taught bioinformatics workshops in
the US, Japan, China, Kenya and once before at GTPB
(<a href="http://gtpb.igc.gulbenkian.pt/bicourses/PHYLOINF09/">PHYLOINF09</a>).
<br>
<br>
<b>Affiliation:</b> <a href="http://naturalis.nl">Naturalis Biodiversity
Center</a>, Leiden, the Netherlands
</td></tr>

<tr><td>
<img border="0" style="margin:4px" src="../images/darin.jpg" width="91"  align="left"></img> 
<font size="2"><p align="justify">
<b>Darin London</b> studied biology at Texas Tech University in the
United States, graduating with a Masters of Science in 1999.  He has been
supporting biological research with computational support for 12 years, starting
at GlaxoSmithKline in 2000, then moving to the European Bioinformatics Institute
to work on the Biomart system, and finally coming to the Institute for Genome
Sciences and Policy at Duke University, Durham, USA in 2005.  He has developed
automated analysis pipelines to help researchers analyze over 25Tb of Vertebrate
NGS data (GAII and HiSeq), mostly involving research work on the Encyclopedia of
DNA Elements (ENCode).  Darin has taught numerous workshops on programming and
automation with the Perl programming language, including the phyloinformatics
workshops at the NESCent, Durham, USA, and the GTPB
(<a href="http://gtpb.igc.gulbenkian.pt/bicourses/PHYLOINF09/">PHYLOINF09</a>).<br>
<br>
<b>Affiliation:</b> <a href="http://www.genome.duke.edu/">Institute for Genome
Sciences and Policy</a>, Duke University
Medical Center, Durham (NC), USA.
</td></tr>
</table>

<br />
<table width="700" align="center" border="1">
<tr><td>
<h2>Course Description</h2>
<h3>Introduction</h3>
Next generation sequencing (NGS) technologies for DNA have resulted in a yet a
bigger deluge of data. Researchers are learning that analysing such data sets is
becoming the bottleneck in their work. In many cases, several steps in these
analyses are fairly generic (e.g. quality control filtering, alignment to
reference sequences, typing) so that off-the-shelf pipelines can be applied. In
other cases, novel research approaches require development of new analysis
pipelines. Either way, all analysis steps should be repeatable and any changes
made to the data (e.g. renaming, annotation, alignment) should be recorded so
that the provenance of the results is clear and inferences are reproducible. In
this brief workshop we will establish several <b>best practices</b> of
reproducibility and provenance recording in the (comparative) analysis of data
obtained by NGS. In doing so we will encounter the commonly used
<b>technologies</b> that enable these best practices by working through <b>use
cases</b> that illustrate the underlying principles.
<h3>Best practices</h3>
<ul>
	<li>Standardized project organization</li>
	<li>Projects fully 'runnable' without user intervention</li>
	<li>No loss of data, metadata or source code through versioning</li>
</ul>
<h3>Technologies</h3>
<ul>
	<li>Next generation sequencing platforms</li>
	<li>File formats</li>
	<li>Command-line executables, command line scripting and batching</li>
	<li>High-level programming with domain-specific toolkits</li>
	<li>Revision control systems</li>
	<li>Workflow environments</li>
</ul>
<h3>Use cases</h3>
<ul>
	<li>Phylogenetic placement of metagenomic data</li>
	<li>Typing of pathogens</li>
	<li>Comparative analysis of multicellular genomic data</li>
</ul>
<h2>Target audience</h2>
This course is aimed at researchers who've recently embarked on NGS projects and
now, faced with large amounts of data, would like to learn how to automate
generic analysis steps and develop new ones in a reproducible (and shareable)
way. The course will be illustrated with examples from specific single-celled
and multicellular taxa but the concepts are applicable to any species not
explicitly referred to in the course. The ideal attendee is a scientist who is
not afraid to get his/her hands dirty to acquire the computer-literacy skills
for dealing with the informatics side of data analysis.
<h2>Pre-requisites</h2>
This course assumes that attendees understand the science behind DNA sequencing
and sequence analysis, and will not go deeply into underlying theory (or the
mechanics of given algorithms, for example) as such. What will be taught are
technical solutions for automating and scaling up such analyses, which will
include beginner-level programming. Some previous experience with running
scripts in any programming language (e.g. R, Perl, Python, batch scripts) will
be helpful, as will general computer literacy (e.g. editing plain text data
files, navigating the command line).
</td></tr>
</table>


<br />

<table width="700" align="center" border="1">
<tr><td>
<br /><a href="application.html"><b>Application</b></a>
</td></tr>

<tr><td>
<br /><a href="timetable.html"><b>Detailed Program</b></a>
</td></tr>

<tr><td>

<br />


</td></tr>



</table>


<br />

<table width="700" align="center" border="1">
<tr><td>
<p>
Instituto Gulbenkian de Ci&ecirc;ncia,</p>
<p>
Apartado 14, 2781-901 Oeiras, Portugal</p>
<p><a href="http://gtpb.igc.gulbenkian.pt/bicourses">GTPB Homepage</a></p>
<p><a href="http://www.igc.gulbenkian.pt">IGC Homepage</a></p>
<p>Last updated:&nbsp;&nbsp;<i><b> July 30th 2012</b></i></p>
</td></tr>
</table>
</body>

</html>
